operational/build.gradle

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+
+
+//plugins {
+  //  id 'com.github.johnrengelman.shadow' version '2.0.3'
+//}
+
+apply plugin: "distribution"
+apply plugin: "java"
+apply plugin: "maven"
+apply plugin: "application"
+apply plugin: "maven-publish"
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
This project does not compile. I recommend removing shadow as it is only required for Flink apps.
```
A problem was found with the configuration of task ':operational:shadowJar'.
No value has been specified for property 'mainClassName'.
```